subprojects/gradle/ci-logger/build.gradle.kts

```diff
@@ -9,4 +9,5 @@ dependencies {
     api(project(":common:logger"))
     implementation(project(":gradle:sentry-config"))
     implementation(project(":gradle:build-environment"))
+    implementation(project(":common:elastic"))
```

Comment:
I see Elastic as one of the `common:logger` implementatios

Reply:
i started with that approach, but it's not like that; elastic is just a transport destination